### PR TITLE
Worked on dashboard display

### PR DESCRIPTION
**Pull request by khanrafay**
- Created a dashboard in which all users are listed down.
- Created details dashboard page where specific user is shown as per id using useParam

### CODE REVIEW

File: src/Pages/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                    <td><Link to={`/dashboard/${user.id}`}>View</Link></td>
+                </tr>
+            )
+
+        })
+        return (
+            <div className="container">
+                <Table striped bordered hover>
+                    <thead>
+                        <tr>
```
Comment (mtahir08) [inline src/Pages/Dashboard.js]: make it a separate component

File: src/Pages/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            .then(response => response.json())
+            .then(response => {
+                setUserDetail([response])
+                setIsLoading(false)
+            })
+    }, [])
+         console.log('response',userDetail)
+    const userData = !isLoading ?  userDetail.map(user => {
+        return (
+            <tr key={user.id}>
```
Comment (mtahir08) [inline src/Pages/Dashboard.js]: make is separate component

File: src/Pages/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+export default Dashboard
+
+
+function UserDetail() {
+    const { userId } = useParams()
+    
+    const [userDetail, setUserDetail] = useState([])
+    const [isLoading,setIsLoading] = useState(false)
+
+    useEffect(() => {
```
Comment (mtahir08) [inline src/Pages/Dashboard.js:75]: make it with redux

**Review by mtahir08 — CHANGES_REQUESTED** [review]

File: src/Pages/Dashboard.js (hunk 1 above)
Comment (khanrafay) [reply src/Pages/Dashboard.js]: Created it.

File: src/Pages/Dashboard.js (hunk 3 above)
Comment (khanrafay) [reply src/Pages/Dashboard.js:75]: Did it.

File: src/Pages/Dashboard.js (hunk 2 above)
Comment (khanrafay) [reply src/Pages/Dashboard.js]: Created...

**Pull request merged**